### PR TITLE
Introduction to C++ operators

### PR DESCRIPTION
**Pull request by Dawe-7**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Dawe-7** [comment]
This fixes #1225

**Comment by Dawe-7** [comment]
This fixes #1197

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/introduction-to-c++-operators/index.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+```bash
+The value of x: 12
+The value of x: 61
+The value of x: 49
+The value of x: -61
+The value of x: 240
+The value of x: 15
+```
+
+In the program above we compare each of the binary digits of the two integers we have and return a 1 with a new integer anywhere both of the binary digits of the two integers had a 1 and a 0 where the two binary digits are not 1. For example in the `&` operator we have:
```
Comment (zolomohan) [inline articles/introduction-to-c++-operators/index.md:350]: This line is too long and difficult to understand. Please explain better.

**Review by zolomohan — CHANGES_REQUESTED** [review]
@Dawe-7 Please make these changes. The explanations in the Bitwise operator section are somewhat harder to understand. Please explain the operations more clearly in a step by step manner.

**Review by zolomohan — APPROVED** [review]

**Comment by hectorkambow** [comment]
@Dawe-7  
👍  Great tutorial topic - is seems like it would be really useful to beginners. 
Please take note of the changes I made to your article for it to flow better.

See what edits and suggestions can be applied to future articles moving forward.

Based on the # of revisions and level of review needed at final - I will be marking this article with a tiered payout label.

Please go over our [resources page](https://github.com/section-io/engineering-education/blob/master/new_contributors/resources-page.md) to see what could help future articles as well.

We encourage all our students to go over the guides as we want to be sure we are presenting our best work to the community and future potential employers.

cc @zolomohan

**Pull request merged**